src/api/store/team.py

```diff
@@ -213,12 +217,16 @@ def update_team(self, team_id, args) -> (dict, MassEnergizeAPIError):
             team.parent = parent
         
         if logo:
-          logo = Media.objects.create(file=logo, name=f"{slugify(team.name)}-TeamLogo")
-          logo.save()
-          team.logo = logo
+          # if existing logo, the string length is around 300 characters
+          # If a new logo updated, this will be the length of the file, much larger than that       
+          new_logo = len(logo) > 1000
```

Comment:
i am confused.  the comment talks of 300 chars and the code dictates 1000

Reply:
That number is arbitrary.  A real image file will be at least a few kilobytes, much larger than 1000.  So 1000 seems like a safe cutoff to test for a photo upload.